src/hotspot/share/code/codeCache.hpp

```diff
@@ -461,4 +466,11 @@ typedef CodeBlobIterator<CompiledMethod, CompiledMethodFilter, true /* is_relaxe
 typedef CodeBlobIterator<nmethod, NMethodFilter, false /* is_relaxed */> NMethodIterator;
 typedef CodeBlobIterator<CodeBlob, AllCodeBlobsFilter, false /* is_relaxed */> AllCodeBlobsIterator;
 
+struct CodeCacheSegment {
```

Comment:
Would this create a confusion?
`CodeHeap` consists of blocks and names them `segments`.  See `src/hotspot/share/memory/heap.hpp` and  `src/hotspot/share/memory/heap.cpp`.
There is `CodeCache::allocated_segments()` which returns the total number of segments (memory blocks) code heaps use. Also there is `CodeCacheSegmentSize` which defines the size of a memory block for a code heap.

Reply:
ok